istioctl/pkg/rbac/testdata/rbac-policies-v2-expected.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.istio.io/v1alpha1
```

Comment:
can we rename this file to `rbac-policies.golden.yaml`?

Reply:
Why is it? Is it like a convention?